### PR TITLE
Enable prometheus pprof new

### PR DESCRIPTION
**Pull request by sonyafenge**
**What type of PR is this?**
> /kind feature

**What this PR does / why we need it**:
Revert Initial code merging kube-up & start-kubemark from scale-out-poc to master #945 because this will failed to start kube-up cluster if ENABLE_APISERVER_INSECURE_PORT=true

Enable insecure-port for kube-apiserver

> set env var KUBE_ENABLE_APISERVER_INSECURE_PORT=true

Add prometheus debug to cluster

> set env var KUBE_ENABLE_APISERVER_INSECURE_PORT=true KUBE_ENABLE_PROMETHEUS_DEBUG=true to enable prometheus debug for kube-apiserver, etcd, kube-controller-manager, kube scheduler

Add pprof collecting for debug purpose

> set env var KUBE_ENABLE_PPROF_DEBUG=true

**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note

```

### CODE REVIEW

File: pkg/kubemark/hollow_kubelet.go [hunk 1]
```diff
@@ -142,6 +142,7 @@ func GetHollowKubeletConfig(opt *HollowKubletOptions) (*options.KubeletFlags, *k
 	c.FileCheckFrequency.Duration = 20 * time.Second
 	c.HTTPCheckFrequency.Duration = 20 * time.Second
 	c.NodeStatusUpdateFrequency.Duration = 10 * time.Second
+	c.NodeStatusReportFrequency.Duration = 5 * time.Minute
```
Comment (pdgetrf) [inline pkg/kubemark/hollow_kubelet.go:145]: thanks @sonyafenge for picking this up.

@Sindica this is to add back the line wrongly removed in https://github.com/CentaurusInfra/arktos/pull/955/files#diff-4ff678479fda4832300707ddef26710d07f76a2415edfc6504045e1c78a8f0bcL144

**Comment by vinaykul** [comment]
Please hold off on the revert. 
Let's apply PR https://github.com/CentaurusInfra/arktos/pull/957 to what you have and see if it addresses the issue. We may need to make some changes to the code to start prometheus with Peng's help.

File: pkg/kubemark/hollow_kubelet.go (hunk 1 above)
Comment (pdgetrf) [reply pkg/kubemark/hollow_kubelet.go:145]: no further comments on this. resolved. (no resolve button here)

Comment (XiaoningDing) [reply pkg/kubemark/hollow_kubelet.go:145]: Help Peng to resolve the comment.

**Pull request closed without merging**